python/make_finemap_inputs.py

```diff
@@ -134,7 +134,7 @@ def read_sumstats(path,
 
     if scale_se_by_pval:
         se = np.abs(sumstats.beta / stats.norm.ppf(sumstats.p.astype(float) / 2))
-        se[(sumstats.beta == 0) | np.isnan(se)] = sumstats.se[(sumstats.beta == 0) | np.isnan(se)]
+        se[(sumstats.beta == 0) | np.isnan(se) | np.isinf(se)] = sumstats.se[(sumstats.beta == 0) | np.isnan(se) | np.isinf(se)]
```

Comment:
```suggestion
        se[(sumstats.beta == 0) | np.isfinite(se)] = sumstats.se[(sumstats.beta == 0) | np.isfinite(se)]
```
I'd suggest replacing `np.isnan` and `np.isinf` with `np.isfinite`. `np.isfinite` exactly do both:
```
Test element-wise for finiteness (not infinity or not Not a Number).
```
https://numpy.org/doc/stable/reference/generated/numpy.isfinite.html

Reply:
replaced with ~np.isfinite

Reply:
ah yes, you're right 👍